admin/api/item/get.js

```diff
@@ -68,7 +68,7 @@ module.exports = function(req, res) {
 									items: _.map(results, function(i) {
 										return {
 											label: refList.getDocumentName(i),
-											href: '/keystone/' + refList.path + '/' + i.id
+											href: '/internal/cms/keystone/' + refList.path + '/' + i.id
```

Comment:
kalo cuma di prepend /internal/cms apa kelebihannya dibanding /keystone biasa aja? soalnya anggapanya kalo mau di harden, kalo url belakangnya di obfuscate we can always assume the hacker can always find out the url?

untuk keystone 0.4.x masih beta jadi kita masih belum berani. Nanti mungkin kita bakalan ambil 0.3.x terus untuk project cms internal (pengganti keystone / kita ngefork keystone sendiri, ini projectnya intern theo pratama)